dialect/sql/builder.go

```diff
@@ -1563,8 +1576,23 @@ func (s *Selector) Table() *SelectTable {
 
 // Join appends a `JOIN` clause to the statement.
 func (s *Selector) Join(t TableView) *Selector {
+	return s.join("JOIN", t)
+}
+
+// LeftJoin appends a `LEFT JOIN` clause to the statement.
+func (s *Selector) LeftJoin(t TableView) *Selector {
+	return s.join("LEFT JOIN", t)
+}
+
+// RightJoin appends a `RIGHT JOIN` clause to the statement.
+func (s *Selector) RightJoin(t TableView) *Selector {
+	return s.join("RIGHT JOIN", t)
+}
+
+// join
+func (s *Selector) join(kind string, t TableView) *Selector {
```

Comment:
Improve comment please.

Reply:
*Join(Join, LeftJoin....) use it together

Reply:
because: https://github.com/facebook/ent/issues/805